src/test/kotlin/no/nb/bikube/core/controller/CoreControllerIntegrationTest.kt

```diff
@@ -337,4 +338,196 @@ class CoreControllerIntegrationTest (
             .expectNextCount(0)
             .verifyComplete()
     }
+
+    @Test
+    fun `search-item endpoint should return 200 OK with all items`() {
+        webClient
+            .get()
+            .uri { uri ->
+                uri.pathSegment("item", "search")
+                    .queryParam("titleCatalogueId", "1")
+                    .queryParam("materialType", MaterialType.NEWSPAPER)
+                    .queryParam("date", "2020-01-01")
+                    .queryParam("isDigital", "true")
+                    .build()
+            }
+            .exchange()
+            .expectStatus().isOk
+            .returnResult<Item>()
+            .responseBody
+            .test()
+            .expectSubscription()
+            .expectNextCount(collectionsModelMockItemA.getObjects()!!.size.toLong())
+            .verifyComplete()
+    }
+
+    @Test
+    fun `search-item endpoint should return mapped items`() {
+        webClient
+            .get()
+            .uri { uri ->
+                uri.pathSegment("item", "search")
+                    .queryParam("titleCatalogueId", "1")
+                    .queryParam("materialType", MaterialType.NEWSPAPER)
+                    .queryParam("date", "2020-01-01")
+                    .queryParam("isDigital", "true")
+                    .build()
+            }
+            .exchange()
+            .returnResult<Item>()
+            .responseBody
+            .test()
+            .expectNext(
+                Item(
+                    catalogueId = collectionsModelMockItemA.getObjects()!![0].priRef,
+                    name = collectionsModelMockItemA.getObjects()!![0].getName(),
+                    date = collectionsModelMockItemA.getObjects()!![0].getItemDate(),
+                    materialType = collectionsModelMockItemA.getObjects()!![0].getMaterialTypeFromParent()!!.norwegian,
+                    titleCatalogueId = collectionsModelMockItemA.getObjects()!![0].getTitleCatalogueId(),
+                    titleName = collectionsModelMockItemA.getObjects()!![0].getTitleName(),
+                    digital = collectionsModelMockItemA.getObjects()!![0].getFormat() == CollectionsFormat.DIGITAL,
+                    urn = collectionsModelMockItemA.getObjects()!![0].getUrn()
```

Comment:
Made an extension function earlier just to make these kinds of tests a tad prettier
```suggestion
                    catalogueId = collectionsModelMockItemA.getFirstObject()!!.priRef,
                    name = collectionsModelMockItemA.getFirstObject()!!.getName(),
                    date = collectionsModelMockItemA.getFirstObject()!!.getItemDate(),
                    materialType = collectionsModelMockItemA.getFirstObject()!!.getMaterialTypeFromParent()!!.norwegian,
                    titleCatalogueId = collectionsModelMockItemA.getFirstObject()!!.getTitleCatalogueId(),
                    titleName = collectionsModelMockItemA.getFirstObject()!!.getTitleName(),
                    digital = collectionsModelMockItemA.getFirstObject()!!.getFormat() == CollectionsFormat.DIGITAL,
                    urn = collectionsModelMockItemA.getFirstObject()!!.getUrn()
```